src/network/NetworkManagerPrimary.hh

```diff
@@ -31,7 +31,7 @@
 #include <gz/sim/Entity.hh>
 #include <gz/transport/Node.hh>
 
-#include "msgs/simulation_step.pb.h"
+#include "gz/sim/private_msgs/simulation_step.pb.h"
```

Comment:
SerializedStateMap, SimulationStep, private_msgs::PerformerAffinity

Reply:
`SerializedStateMap` is in `serialized_map.pb.h`